pages/documentation/migration-V3.md

```diff
@@ -512,7 +533,7 @@ Setting | Old default | New default
 We deprecated the `$theme-site-max-width` variable. We're using `$theme-grid-container-max-width` instead.
 
 ###### What to do
-1. Replace instances of `$theme-site-max-width` with `$theme-grid-container-max-width`
+1. Replace instances of `$theme-site-max-width` with `$theme-grid-container-max-width`. If your layout is affected by this change, remove this setting from your settings file.
```

Comment:
Add a note about checking to see if there are any regressions after replacing this setting